src/app/workspace/cmdinput/aichat.tsx

```diff
@@ -110,12 +111,14 @@ class AIChat extends React.Component<{}, {}> {
     onTextAreaFocused(e: any) {
         mobx.action(() => {
             this.isFocused.set(true);
```

Comment:
Do we need a local isFocused state? We should just be able to use the auxViewFocus global var to track this for all the views

Reply:
I don't think we do, addressed